packages/opentelemetry/src/semanticAttributes.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * These are internal and are not supposed to be used/depended on by external parties.
+ * No guarantees apply to these attributes, and the may change/disappear at any time.
+ */
+export const InternalSentrySemanticAttributes = {
```

Comment:
Instead of constants, keeping this similar to how OTEL handles their semantic attributes. Eventually we should align this with https://github.com/getsentry/rfcs/pull/116.